exporters/jaeger/src/main/java/io/opentelemetry/exporter/jaeger/JaegerGrpcSpanExporterBuilder.java

```diff
@@ -35,13 +44,34 @@ public JaegerGrpcSpanExporterBuilder setChannel(ManagedChannel channel) {
   }
 
   /**
-   * Sets the Jaeger endpoint to connect to. Optional, defaults to "localhost:14250".
-   *
-   * @param endpoint The Jaeger endpoint URL, ex. "jaegerhost:14250".
-   * @return this.
+   * Sets the Jaeger endpoint to connect to. If unset, defaults to {@value DEFAULT_ENDPOINT_URL}.
+   * The endpoint must start with either http:// or https://.
    */
   public JaegerGrpcSpanExporterBuilder setEndpoint(String endpoint) {
-    this.endpoint = endpoint;
+    requireNonNull(endpoint, "endpoint");
+    if (!endpoint.contains("://")) {
+      endpoint = "http://" + endpoint;
+    }
+
+    URI uri;
+    try {
+      uri = new URI(endpoint);
+    } catch (URISyntaxException e) {
+      throw new IllegalArgumentException("Invalid endpoint, must be a URL: " + endpoint, e);
+    }
+    // TODO(anuraaga): Remove after announcing deprecation of schemaless URLs.
+    if (uri.getScheme() != null) {
+      if (!uri.getScheme().equals("http") && !uri.getScheme().equals("https")) {
+        throw new IllegalArgumentException("Invalid scheme, must be http or https: " + endpoint);
+      }
+    } else {
+      logger.log(
+          Level.WARNING,
```

Comment:
should we also log this, or something similar, up on line 53 when we have to insert the scheme?